src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
Actually, we spell `left` and `right` when specifying text anchors, so let's go with:

direction: 'left'|'right'|'u['|'down'